AG33220A/iocBoot/iocAG33220A-IOC-02/st.cmd

```diff
@@ -0,0 +1,18 @@
+#!../../bin/windows-x64-debug/AG33220A-IOC-02
+
+## You may have to change AG33220A-IOC-02 to something else
+## everywhere it appears in this file
+
+# Increase this if you get <<TRUNCATED>> or discarded messages warnings in your errlog output
+errlogInit2(65536, 256)
+
+< envPaths
+
+cd "${TOP}"
+
+## Register all support components
+dbLoadDatabase "dbd/AG33220A-IOC-02.dbd"
+AG33220A_IOC_02_registerRecordDeviceDriver pdbbase
+
+## calling common command file in ioc 01 boot dir
+< ${TOP}/iocBoot/ioc_01_APP_NAME_/st-common.cmd
```

Comment:
Needs filling in. This means the ioc can't boot and the ioc tests for ioc _02 fail.

This is also true for all of the other IOCs in this PR.